web/src/components/tasks/LicenseStatusReceipt.tsx

```diff
@@ -127,27 +122,24 @@ export const LicenseStatusReceipt = (props: Props): ReactElement => {
       return "";
     }
     const { nameAndAddress } = business.licenseData;
-
     const secondLineAddress = nameAndAddress.addressLine2 ? ` ${nameAndAddress.addressLine2}` : "";
-
     return `${nameAndAddress.addressLine1}${secondLineAddress}, ${nameAndAddress.zipCode} NJ`;
   };
 
-  const getIconColor = (item: LicenseStatusItem): string => {
-    if (props.status === "ACTIVE" || props.status === "PENDING") {
-      return item.status === "ACTIVE" ? activePermitTheme.iconColor : pendingPermitTheme.iconColor;
-    } else {
-      return item.status === "ACTIVE" ? grayPermitTheme.iconColor : pendingPermitTheme.iconColor;
-    }
-  };
-
-  const receiptItem = (item: LicenseStatusItem, index: number): ReactElement => {
-    const border = index === 0 ? "" : "border-top-1px border-base-light";
-
+  const receiptItem = (
+    item: LicenseStatusItem,
+    index: number,
+    receiptItems: LicenseStatusItem[]
+  ): ReactElement => {
     return (
-      <div className="padding-x-3 fdr fac width-100" data-testid={`item-${item.status}`} key={index}>
-        <Icon className={`${getIconColor(item)} usa-icon--size-3`}>{getIcon(item.status)}</Icon>
-        <span className={`margin-left-2 padding-y-1 fg1 receipt-item ${border}`}>{item.title}</span>
+      <div key={index} data-testid={`item-${item.status}`}>
+        <div className="flex flex-column fac tfr width-full pt-1 tpt-0">
+          <ChecklistTag status={item.status} />
+          <span className="margin-left-2 text-left width-full margin-top-1 tablet:margin-top-0">
+            {item.title}
+          </span>
+        </div>
+        {index === receiptItems.length - 1 ? <></> : <hr className="desktop:margin-bottom-1" />}
```

Comment:
[pebble] Could we wrap this div in another outer div and just add `className="desktop:margin-bottom-1"` so it's easier to read and doesn't require index math?

Reply:
@FarazA22 I misread this, yeah that makes sense!